crates/biome_css_analyze/tests/specs/correctness/noUnmatchableAnbSelector/valid.scss

```
@@ -0,0 +1,6 @@
+/* should not generate diagnostics */
+a:nth-child(#{$i}) {}
```

Stylelint (17.9.0)

[error] Empty block (block-no-empty)

(block-no-empty)